src/twisted/cred/credentials.py

```diff
@@ -33,7 +33,7 @@ class ICredentials(Interface):
     I check credentials.
 
     Implementors I{must} specify the sub-interfaces of ICredentials
-    to which it conforms, using L{zope.interface.declarations.implementer}.
```

Comment:
Is this change necessary?  `implementer` is actually defined in `zope.interface.declarations`
although `zope.interface` imports it.

Reply:
It is defined in `declarations`, but not documented there, so to link to its documentation we need to point to the parent module.

Reply:
Ah, OK.